pages/invest/index.js

```diff
@@ -0,0 +1,47 @@
+import Link from 'next/link';
+import { Card } from '@components/stock-card';
+import { Footer } from '@components/footer';
+import stockData from '../../mock/stocks.json';
+import personData from '../../mock/person.json';
+import NavBar from '@components/NavBar';
+
+const stock = Object.keys(stockData);
+const Invest = () => {
+  return (
+    <>
+      <NavBar personData={personData} />
+      <div className="main-container">
+        <div className="layout">
+          <div className="content">
+            <div className="shoppinglist-container">
+              {stock.map((itemName) => {
+                return <Card key={itemName} stock={stockData[itemName]} />;
+              })}
```

Comment:
since you are returning only 1 line of code, you can transform this like this
```javascript
{stock.map((itemName) => <Card key={itemName} stock={stockData[itemName]} />)}
```

Reply:
I will do the changes. Thanks for the review😀

Reply:
May consider below to please the human eyes 
`{ stock.map( itemName => <Card key = {itemName} stock = { stockData[itemName] } />) }`